contrib/devtools/update-translations.py

```diff
@@ -122,6 +124,12 @@ def escape_cdata(text):
     text = text.replace('"', '&quot;')
     return text
 
+def contains_bitcoin_addr(text, errors):
+    if text != None and ADDRESS_REGEXP.search(text) != None:
```

Comment:
Could be:
```py
if ADDRESS_REGEXP.search(text):
    ...
```

Reply:
If someone else feels strongly about making the change here, lmk and I will make it. If it's not necessary, don't want to reset the ACKs based off a new commit hash :)